docs/querying/sql.md

```diff
@@ -376,6 +376,7 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
 
 |Function|Notes|
 |--------|-----|
+|`PI`|Constant Pi.|
```

Comment:
```suggestion
|`PI()`|Constant Pi.|
```

Reply:
Just like `CURRENT_TIMESTAMP`, this function not need a `()`. Actually, sql will not work when write PI as `PI()`.